cli/tauri.js/src/types/cargo.ts

```diff
@@ -1,5 +1,10 @@
+export interface CargoTomlDependency {
+  version?: string
+  path?: string
+}
+
 export interface CargoManifest {
-  dependencies: { [k: string]: string | CargoManifestDependency }
+  dependencies: { [k: string]: CargoManifestDependency }
```

Comment:
shouldn't remove the string type option here, the dependency can be just its string version